### PR TITLE
DEV: Add vbulletin5 bulk importer

### PR DESCRIPTION
**Pull request by justindirose**
This is a pretty straightforward bulk importer, just tailored to the vBulletin 5 database structure. 

Also made a few minor improvements to the base importer -- should be self explanatory in the code.

### CODE REVIEW

File: script/bulk_import/base.rb [hunk 1]
```diff
@@ -639,12 +643,12 @@ def process_raw(original_raw)
     # }
 
     # [QUOTE=<username>;<postid>]
-    raw.gsub!(/\[QUOTE=([^;\]]+);(\d+)\]/i) do
+    raw.gsub!(/\[QUOTE=([^;\]]+);n(\d+)\]/i) do
```
Comment (ZogStriP) [inline script/bulk_import/base.rb:646]: Was that `n` intended?

File: script/bulk_import/vbulletin5.rb [hunk 2]
```diff
@@ -0,0 +1,779 @@
...
+      username: username,
+      password: password,
+      database: database,
+      encoding: charset,
+      reconnect: true
+    )
+
+    @client.query_options.merge!(as: :array, cache_rows: false)
+
+    @forum_typeid = mysql_query("SELECT contenttypeid FROM #{DB_PREFIX}contenttype WHERE class='Forum'").to_a[0][0]
```
Comment (ZogStriP) [inline script/bulk_import/vbulletin5.rb:43]: Nitpick, but you could `LIMIT 1` since you're interested in only 1 result

**Review by ZogStriP — APPROVED** [review]

File: script/bulk_import/base.rb (hunk 1 above)
Comment (justindirose) [reply script/bulk_import/base.rb:646]: It likely was. For this DB the `n` didn't exist in the quotes. I'll revert the change.

File: script/bulk_import/vbulletin5.rb (hunk 2 above)
Comment (justindirose) [reply script/bulk_import/vbulletin5.rb:43]: It's worth leaving as a note -- I don't have a great way to test the change at this time.

**Pull request merged**